ansible/roles/openslide/tasks/main.yml

```diff
@@ -88,14 +90,19 @@
 
 # libtiff
 - name: Download libtiff
-  get_url:
-    url: http://download.osgeo.org/libtiff/tiff-4.0.6.tar.gz
-    dest: "{{ root_dir }}"
+
+  command: >-
+    wget --retry-connrefused --waitretry=1 --read-timeout=20 http://download.osgeo.org/libtiff/tiff-4.0.6.tar.gz
```

Comment:
osgeo sometimes takes 100 seconds to respond.   Can we increase this to something even larger (perhaps 300 seconds)?